Functions/Example/FirebaseFunctions.xcodeproj/project.pbxproj

```diff
@@ -494,13 +500,8 @@
 			isa = XCBuildConfiguration;
 			buildSettings = {
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
-				GCC_PRECOMPILE_PREFIX_HEADER = NO;
 				INFOPLIST_FILE = "FirebaseFunctions/FirebaseFunctions-Info.plist";
 				MODULE_NAME = ExampleApp;
-				OTHER_LDFLAGS = (
-					"$(inherited)",
-					"-all_load",
```

Comment:
-all_load should stay to catch more errors earlier

Reply:
Re-added in latest commit.